apps/src/javalab/JavalabButton.jsx

```diff
@@ -64,19 +65,9 @@ JavalabButton.defaultProps = {
 };
 
 const styles = {
-  button: {
-    display: 'flex',
-    flexDirection: 'column',
-    alignItems: 'center',
-    color: color.white
-  },
-  horizontal: {
-    flexDirection: 'row'
-  },
   disabled: {
     backgroundColor: color.light_gray,
     borderColor: color.light_gray,
```

Comment:
Could this not also be configured with a class name? Could you pass in some sort of `.disabled` class name to override these values?

Reply:
Some more discussion about how CSS module class precedence here and how it relates to disabled styling:

https://github.com/code-dot-org/code-dot-org/pull/47631/files#r950423986

I think one solution would be that each consumer could pass in its own disabled styling to JavalabButton, but that doesn't feel great to me (I'd rather have JavalabButton control that). LMK if you have other ideas.